### PR TITLE
Add docs for android docker image.

### PR DESCRIPTION
**Pull request by jrbenny35**
https://mozilla-hub.atlassian.net/browse/ECTEEN-81

### CODE REVIEW

File: docs/android.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Klaatu for Android
+
+The preferred method of running the android tests are using the provided docker image. The steps below outline how that can be done.
+
+## Docker
+
+Close your android studio instance if it is running. Navigate to your `platform-tools` folder where `adb` is located and run the following command:
+
+```sh
+./adb.exe -a nodaemon server start
```
Comment (pdehaan) [inline docs/android.md:10]: Does this work on Linux/macOS too? (re: that `.exe` extension).

File: docs/android.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Klaatu for Android
+
+The preferred method of running the android tests are using the provided docker image. The steps below outline how that can be done.
```
Comment (pdehaan) [inline docs/android.md:3]:
```suggestion
The preferred method of running the Android tests are using the provided docker image. The steps below outline how that can be done.
```

File: docs/android.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Docker
+
+Close your android studio instance if it is running. Navigate to your `platform-tools` folder where `adb` is located and run the following command:
```
Comment (pdehaan) [inline docs/android.md:7]:
```suggestion
Close your Android Studio instance if it is running. Navigate to your `platform-tools` folder where `adb` is located and run the following command:
```

**Review by pdehaan — APPROVED** [review]

File: docs/android.md (hunk 1 above)
Comment (jrbenny35) [reply docs/android.md:10]: Good catch, lemme fix that

**Pull request merged**